### ISSUE TITLE
Azure Resource Manager (ARM) Tools tries to download .NET Core Runtime version 3.0.2 that is no longer supported

### ISSUE THREAD

**Issue by enzanglach** [microsoft/vscode-azurearmtools#612]
Does this occur consistently? Yes
Repro steps:
1. Install Azure Resource Manager (ARM) Tools msazurermtools.azurerm-vscode-tools plugin on VS Code
2. Open ARM template

Action: acquireDotnet
Error type: 1
Error Message: >>>> This extension requires .NET Core for full functionality, but we were unable to download and install a local copy for the extension. If this error persists, please see https://aka.ms/vscode-armtools-dotnet for troubleshooting tips. 
Command failed: "c:\Users\sms85451\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.9.0\assets\install scripts\dotnet-install.cmd" -InstallDir 'c:\Users\sms85451\AppData\Roaming\Code\User\globalStorage\msazurermtools.azurerm-vscode-tools\.dotnet\3.0.2' -Runtime dotnet -Version 3.0.2
Could not find/download: ".NET Core Runtime" with version = 3.0.2
Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
At C:\Users\sms85451\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.9.0\assets\install scripts\dotnet-install
.ps1:656 char:5
+     throw "Could not find/download: `"$assetName`" with version = $Sp ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find/...ET Core support:String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find/download: ".NET Core Runtime" with version = 3.0.2
Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
 

Version: 0.9.0
OS: win32
OS Release: 10.0.17134
Product: Visual Studio Code
Product Version: 1.44.1
Language: en

<details>
<summary>Call Stack</summary>

```
ChildProcess.exithandler child_process.js:308:12
ChildProcess.emit events.js:203:13
maybeClose child_process.js:1021:16
Process.ChildProcess._handle.onexit child_process.js:283:5
```

</details>

<details>
<summary>dotnetAcquireLog</summary>

```
Status bar: $(cloud-download) Downloading .NET Core tooling...
Status bar: $(alert) Error acquiring .NET Core tooling!

[dotnet 3.0.2]: Found an incompletely-installed version of dotnet 3.0.2... Uninstalling.
[dotnet 3.0.2]: Uninstalling dotnet version 3.0.2
[dotnet 3.0.2]: Removing folder c:\Users\sms85451\AppData\Roaming\Code\User\globalStorage\msazurermtools.azurerm-vscode-tools\.dotnet\3.0.2
[dotnet 3.0.2]: Finished removing folder c:\Users\sms85451\AppData\Roaming\Code\User\globalStorage\msazurermtools.azurerm-vscode-tools\.dotnet\3.0.2
[dotnet 3.0.2]: Finished uninstalling dotnet version 3.0.2
Using this command to install .NET Core version 3.0.2:
"c:\Users\sms85451\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.9.0\assets\install scripts\dotnet-install.cmd" -InstallDir 'c:\Users\sms85451\AppData\Roaming\Code\User\globalStorage\msazurermtools.azurerm-vscode-tools\.dotnet\3.0.2' -Runtime dotnet -Version 3.0.2
Downloading .NET Core tooling version(s) 3.0.2...........[dotnet 3.0.2]: dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Runtime/3.0.2/dotnet-runtime-3.0.2-win-x64.zip
dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Runtime/3.0.2/dotnet-runtime-3.0.2-win-x64.zip
dotnet-install: Downloading legacy link: https://dotnetcli.azureedge.net/dotnet/Runtime/3.0.2/dotnet-win-x64.3.0.2.zip
dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Runtime/3.0.2/dotnet-win-x64.3.0.2.zip

[dotnet 3.0.2]: STDERR: Could not find/download: ".NET Core Runtime" with version = 3.0.2
Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
At C:\Users\sms85451\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.9.0\assets\install scripts\dotnet-install
.ps1:656 char:5
+     throw "Could not find/download: `"$assetName`" with version = $Sp ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find/...ET Core support:String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find/download: ".NET Core Runtime" with version = 3.0.2
Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
 

 Error!
Failed to download .NET Core tooling 3.0.2:
Exit code: 1
Message: Command failed: "c:\Users\sms85451\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.9.0\assets\install scripts\dotnet-install.cmd" -InstallDir 'c:\Users\sms85451\AppData\Roaming\Code\User\globalStorage\msazurermtools.azurerm-vscode-tools\.dotnet\3.0.2' -Runtime dotnet -Version 3.0.2
Could not find/download: ".NET Core Runtime" with version = 3.0.2
Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
At C:\Users\sms85451\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.9.0\assets\install scripts\dotnet-install
.ps1:656 char:5
+     throw "Could not find/download: `"$assetName`" with version = $Sp ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find/...ET Core support:String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find/download: ".NET Core Runtime" with version = 3.0.2
Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
 



```

</details>

**Comment by StephenWeatherford** [comment]
https://github.com/microsoft/vscode-azurearmtools/pull/620

**Comment by StephenWeatherford** [comment]
Thanks for reporting, we just released 0.9.1, which we believe will fix this issue.  Please let us know if this works for you.
